### DISCUSSION TITLE
Reordering is not allowed for this resource.

### DISCUSSION THREAD

**Discussion by AndreyPirozhkov** [outl1ne/nova-sortable#155] [Q&A]
I did everything according to the instructions https://github.com/outl1ne/nova-sortable/blob/3.2.1/docs/sorting/many-to-many.md
But the navigation buttons are not available.

![image](https://user-images.githubusercontent.com/113905492/194092915-b41bf7a2-0819-42b4-8335-bbf0a4f6ef53.png)

**Comment by KasparRosin** [comment]
Unable to reproduce with the vanilla nova setup and following the default documentation.
Most likely you have some configuration issue.

Could you post your $sortable array.
